pylint/extensions/comparetozero.py

```diff
@@ -46,15 +48,16 @@ class CompareToZeroChecker(checkers.BaseChecker):
     options = ()
 
     @utils.check_messages("compare-to-zero")
-    def visit_compare(self, node):
+    def visit_compare(self, node: nodes.Compare) -> None:
         _operators = ["!=", "==", "is not", "is"]
         # note: astroid.Compare has the left most operand in node.left
         # while the rest are a list of tuples in node.ops
         # the format of the tuple is ('compare operator sign', node)
         # here we squash everything into `ops` to make it easier for processing later
         ops = [("", node.left)]
         ops.extend(node.ops)
-        ops = list(itertools.chain(*ops))
+        iter_ops: Iterable[Any] = iter(ops)
+        ops = list(itertools.chain(*iter_ops))
```

Comment:
Why create a temporary variable?

Reply:
Mypy kept complaining about `*ops` not being an `Iterable` but a `List`. I tried other solutions, but this was the only one I could come up with that satisfied both mypy and the tests.

Same for the other one where I do this.